cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -25,6 +25,19 @@ class CxTowerPlanLine(models.Model):
         auto_join=True,
         ondelete="cascade",
     )
+    related_plan_line_ids = fields.One2many(
+        comodel_name="cx.tower.plan.line",
+        compute="_compute_related_plan_line_ids",
+        string="Related Flight Plan Lines",
+        readonly=True,
+    )
+    related_flight_plan_id = fields.Many2one(
+        comodel_name="cx.tower.plan",
+        compute="_compute_related_flight_plan_id",
```

Comment:
You can make this field related instead of computed. And remove the prefix, there is no another flight_plan_id in this model. Use the same name as in the destination related model.